modules/cli/src/main/scala/scala/cli/launcher/LauncherCLI.scala

```diff
@@ -0,0 +1,59 @@
+package scala.cli.launcher
+import dependency._
+
+import scala.build.internal.{OsLibc, Runner}
+import scala.build.options.{BuildOptions, JavaOptions}
+import scala.build.{Artifacts, Positioned}
+import scala.cli.commands.LoggingOptions
+
+object LauncherCLI {
+
+  def runAndExit(version: String, options: LauncherOptions, remainingArgs: Seq[String]) = {
```

Comment:
NIT This can explicitly be typed as returning `Nothing` (that makes it clearer that this method doesn't really return)
```suggestion
  def runAndExit(version: String, options: LauncherOptions, remainingArgs: Seq[String]): Nothing = {
```